DevelopGitHub/GitHubExtension/GitHubExtension.Security.DAL/Package/DALSecurityPackage.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace GitHubExtension.Security.DAL.Package
 {
-    public class DALPackage : IPackage
+    public class DALSecurityPackage : IPackage
```

Comment:
There are two packages in security module, in future will be one and renamed. Also SimpleInjectorConfiguration will move from Security to Infrastructure, but in another pull request.